.travis.yml

```diff
@@ -51,13 +59,19 @@ after_success:
 
 before_cache: Rscript -e 'remotes::install_cran("pkgdown")'
 deploy:
-  provider: script
-  script: Rscript -e 'pkgdown::deploy_site_github()'
-  skip_cleanup: true
-  on:
-    branch: master
-    condition: "$AFTER_JOBS = true"
-    type: push
+  # our site to gh_pages
+  - provider: script
+    script: Rscript -e 'pkgdown::deploy_site_github()'
+    skip_cleanup: true
+    on:
+      branch: master
+      condition: "$AFTER_JOBS = true"
+      type: push
+  # our image to dockerhub
+  - provider: script
+    script: docker_push
+    on:
+      branch: master
```

Comment:
Add `condition: "$AFTER_JOBS = true"` so it will only deploy from one of the machines.